src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+
+  message.style.top = `${posTop}px`;
+  message.style.right = `${posRight}px`;
+
+  message.classList.add('notification', type);
+
+  const titleElement = document.createElement('h2');
+
+  titleElement.classList.add('title');
+  titleElement.textContent = title;
+  message.appendChild(titleElement);
+
+  const descriptionElement = document.createElement('p');
+
+  descriptionElement.textContent = description;
+  message.appendChild(descriptionElement);
+
+  document.body.appendChild(message);
+
+  setTimeout(() => {
+    message.style.visibility = 'hidden';
```

Comment:
Instead of setting `message.style.visibility = 'hidden';`, consider using `message.remove();` to completely remove the notification from the DOM after the timeout. This will help in managing the DOM elements more efficiently and prevent unnecessary elements from accumulating in the DOM.